Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM haskell:8.6
```

Comment:
If you want to you can split this Dockerfile into two parts, one part that builds the project and one part that runs the project. 

Docs about this: 
https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
I've upgraded GHC to version 8.10, so I think things will work better if you base your image on `haskell:8.10` instead.

Reply:
Should be fixed as of 324c0b41e585f70a41711bc46fd0b31444e46545

Reply:
Awesome! :D